src/service/QuestionService.js

```diff
@@ -31,6 +31,12 @@ export class QuestionService {
                 answers: [ answersObj[0].photoUrl, answersObj[1].photoUrl, answersObj[2].photoUrl, answersObj[3].photoUrl ], 
                 correctAnswer: { value: answersObj[this.correctAnswerIndex].photoUrl, index: this.correctAnswerIndex }
             })
+        } else if (mode === WHO_IS_THAT_POKEMON_HARD_MODE) {
+            return shuffleAnswers({
+                question: answersObj[this.correctAnswerIndex].photoUrl , 
+                answers: [ answersObj[0].name, answersObj[1].name, answersObj[2].name, answersObj[3].name ], 
+                correctAnswer: { value: answersObj[this.correctAnswerIndex].name, index: this.correctAnswerIndex}
+            })
```

Comment:
isn't this code the same as in first mode?
you can use `mode === X || mode === Y` 😌